lib/transmission-rss/client.rb

```diff
@@ -56,6 +56,10 @@ def add_torrent(file, type = :url, options = {})
           raise ArgumentError.new('type has to be :url or :file.')
       end
 
+      if options[:labels]
+        arguments.labels = options[:labels]
+      end
```

Comment:
please add a test

Reply:
Added a test that the value is an array.